### DISCUSSION TITLE
Hello, everyone, does minio has a migrate tool

### DISCUSSION THREAD

**Discussion by WyattJia** [minio/minio#12578] [Q&A] [answered]
We have a batch minio server deploy on raid5 machines with hundreds TB datas. We need migrate those data to another raid0 servers. So we need a high performance migrate tool to transfer data to new machine.

**Comment by harshavardhana** [comment, marked as answer]
you can use `mc mirror` @WyattJia

**Reply by WyattJia** [reply]
Thanks.

**Reply by WyattJia** [reply]
Thanks for answering, but I tried the `mc mirror` tool, we found that the speed of this tool is a bit slow, we have a lot of online data, and it can't affect the online service, so I want to ask if there are other tools to replace or use `rsync` all right?@harshavardhana

**Reply by harshavardhana** [reply]
mc mirror is the best available.. you can use another alternative `rclone`
